lib/qmp.ml

```diff
@@ -149,7 +149,12 @@ let message_of_string str =
 
   let event json =
     let event_data event data =
-      let rtc_offset data = data |> U.member "offset" |> U.to_int in
+      let to_int64 = function
+        | `Intlit i -> Int64.of_string i
+        | `Int i -> Int64.of_int i
+        | _ -> failwith "Got none int or int64"
```

Comment:
I think it would be better to be more explicit: `unknown -> failwith (Printf.sprintf "Unable to convert %s to int64" (Y.to_string unknown))`

Reply:
Updated, thanks.